source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -306,6 +306,81 @@ message AccessChainClampingInfo {
 
 }
 
+message InstToIdsForEnclosing {
```

Comment:
I really like the way you have factored this out, and the comments are great.  However I don't like this name.

I'm finding it hard to think of a good name, but I think it should be of the form SomethingInfo (to mirror AccessChainClampingInfo and LoopLimiterInfo).

How about SideEffectEnclosureInfo, or SideEffectWrapperInfo?

Reply:
I renamed it to SideEffectWrapperInfo